test/pylib/cpp/boost/boost_facade.py

```diff
@@ -100,6 +100,7 @@ def read_file(name: Path) -> str:
         log_xml = root_log_dir / f"{test_name}.log"
         report_xml = root_log_dir / f"{test_name}.xml"
         args = [ str(executable),
+                 '--report_level=no',
```

Comment:
What kind of report is suppressed with that option? And since you say
> ... we don't need a report at the end of the run.
why is e.g. `--report_sink` option is still here? Is it some other report?

Reply:
You are right, it's not needed anymore.